DisCatSharp/DiscordConfiguration.cs

```diff
@@ -210,6 +211,13 @@ public UdpClientFactoryDelegate UdpClientFactory
         /// </summary>
         public bool AutoRefreshChannelCache { internal get; set; } = false;
 
+        /// <summary>
+        /// <para>Sets the service provider.</para>
+        /// <para>This allows passing data around without resorting to static members.</para>
+        /// <para>Defaults to null.</para>
```

Comment:
From what I see, it doesn't look like you set null to this

Reply:
Oh, I didn't notice. I copied this from ApplicationCommands without changing, so [it's the same there now](https://github.com/Aiko-IT-Systems/DisCatSharp/blob/main/DisCatSharp.ApplicationCommands/ApplicationCommandsConfiguration.cs#L36).